docs/source/routing/graphos-features.mdx

```diff
@@ -1,187 +1,35 @@
+## GraphOS Router licensed plan features
 
-- Real-time updates via [GraphQL subscriptions](/graphos/routing/operations/subscriptions)
-- Improved performance with [query batching](/graphos/routing/performance/query-batching)
 - Authentication of inbound requests via [JSON Web Token (JWT)](/graphos/routing/security/jwt)
 - [Authorization of specific fields and types](/graphos/routing/security/authorization) through the [`@requiresScopes`](/graphos/routing/security/authorization#requiresscopes), [`@authenticated`](/graphos/routing/security/authorization#authenticated), and [`@policy`](/graphos/routing/security/authorization#policy) directives
 - Incremental migration between subgraphs through [the progressive `@override` directive](/graphos/schema-design/federated-schemas/entities/migrate-fields#incremental-migration-with-progressive-override).
 - Redis-backed [distributed caching of query plans and persisted queries](/graphos/routing/performance/caching/distributed)
-- Redis-backed [entity caching of subgraph responses](/graphos/routing/performance/caching/entity)
 - Custom request handling in any language via [external coprocessing](/graphos/routing/customization/coprocessor)
 - Mitigation of potentially malicious requests via [request limits](/graphos/routing/security/request-limits),[demand control](/graphos/routing/security/request-limits), and [safelisting](/graphos/routing/security/persisted-queries)
 - Custom instrumentation and telemetry, including [custom attributes for spans](/graphos/reference/router/telemetry/instrumentation/spans#attributes).
 - An [offline license](/graphos/reference/router/graphos-features#the-enterprise-license) that enables running the router with GraphOS features when disconnected from the internet.
 - [Using contexts to share data](/graphos/schema-design/federated-schemas/entities/use-contexts) with the `@context` and `@fromContext` directives
 
-For details on these features, see [this blog post](https://blog.apollographql.com/apollo-router-v1-12-improved-router-security-performance-and-extensibility) in addition to the documentation links above.
+## Enabling licensed plan features
 
-### Enabling plan-specific features
-
-To enable support for plan-specific features in GraphOS Router:
+To enable licensed plan features in a router:
 
 - You must run GraphOS Router v1.12 or later. [Download the latest version.](/graphos/reference/router/self-hosted-install#1-download-and-extract-the-router-binary)
     - Certain features might require a later router version. See a particular feature's documentation for details.
 - Your router instances must connect to GraphOS with a **graph API key** and **graph ref** associated with your organization.
     - You connect your router to GraphOS by setting [these environment variables](/graphos/reference/router/configuration#environment-variables) when starting the router.
     - If your router _already_ connects to your GraphOS organization, no further action is required.
```

Comment:
Will it be clear to users how a router might already be connected to GraphOS?

Reply:
deleting this line